account_banking_mandate/views/res_partner_bank_view.xml

```diff
@@ -15,6 +15,7 @@
                         name="mandate_ids"
                         context="{'mandate_bank_partner_view': True}"
                         nolabel="1"
+                        colspan="4"
```

Comment:
```suggestion
                        colspan="2"
```

Reply:
This is the result with colspan="2"
![image](https://github.com/OCA/bank-payment/assets/45785416/d3cdd97d-ef9b-495e-a053-a75a58cf3fca)